spring-boot/admin/src/main/resources/templates/admin/fragments/api-keys/consumer/list.html

```diff
@@ -1,36 +1,15 @@
 <!DOCTYPE html >
 <html lang="en" xmlns:th="http://www.thymeleaf.org">
     <body>
-        <div id="infoDialog" title="Info">
+        <div id="infoDialog" title="Info" style="display: none">
```

Comment:
`infoDialog` is quite generic, we should try to find a better name. Depending on its use, e.g.:

- `confirmDeletionDialog` in case it is only used for this purpose
- `confirmationDialog` in case there are different kinds of confirmations

The attribute `title` should also be changed. Instead of "Info" it should say e.g. "Please confirm" or "Are you sure?"

Reply:
Another question to be discussed, should we move this into a fragment?

Reply:
I refactored the names but have not applied additional fragmentation yet.

Reply:
Done. I created a fragment for the ```confirmDeletionDialog```.